### PR TITLE
ICP-10646, CHAD-3610 Remove componentLabel and componentName from non-MCD addChildDevice calls

### PR DESCRIPTION
**Pull request by dkirker**

### CODE REVIEW

**Comment by dkirker** [comment]
Added "Work In Progress" label while I test this, and verify if there is a way to update the componentName for existing devices.

File: devicetypes/smartthings/zwave-dual-switch.src/zwave-dual-switch.groovy [hunk 1]
```diff
@@ -60,7 +60,7 @@ def installed() {
 		String dni = "${device.deviceNetworkId}-ep2"
 		addChildDevice("Z-Wave Binary Switch Endpoint", dni, device.hub.id,
 			[completedSetup: true, label: "${componentLabel}",
-			 isComponent: false, componentName: "ch2", componentLabel: "${componentLabel}"])
+			 isComponent: false])
```
Comment (tpmanley) [inline devicetypes/smartthings/zwave-dual-switch.src/zwave-dual-switch.groovy]: nit but could move this up to the previous line since it's so short now

**Review by tpmanley — DISMISSED** [review]

**Comment by tpmanley** [comment]
Since this is affecting users today I think we should get the necessary approvals to release this next week assuming it's passed QA

**Comment by dkirker** [comment]
@tpmanley Yeah, I am trying to wrap up verification today and get it over to QA. The Z-Wave devices were misbehaving, so I'll probably try another hub...

**Review by tpmanley — APPROVED** [review]

**Comment by dkirker** [comment]
@tpmanley Alright! This should be in staging tomorrow.

**Pull request merged**